docs/source/stability_policy.rst

```diff
@@ -0,0 +1,132 @@
+Stability Policy
```

Comment:
You uppercased policy everywhere, I wouldn't, its just a noun no?

Reply:
Don't know why I did that ...